### PR TITLE
Update Operator Framework exercises to use Operator-SDK 1.0.1

### PR DESCRIPTION
**Pull request by madorn**
Update Operator Framework exercises to use Operator-SDK 1.0.1

### CODE REVIEW

**Comment by madorn** [comment]
Please review/merge.

We are trying to get these exercises merged in time for live workshop next week, see https://github.com/openshift-labs/learn-katacoda/issues/903

File: ansibleop/ansible-operator-overview/step5.md [hunk 1]
```diff
@@ -1,95 +1,161 @@
-Next, we'll learn how to use the operator-sdk CLI to create an Ansible Operator project with the 'new' command.
+Next, we'll learn how to use the operator-sdk CLI to create an Ansible Operator project with the 'init' and `create` commands.
```
Comment (btannous) [inline ansibleop/ansible-operator-overview/step5.md:1]: typo. 
```
`init` and `create`
```

File: ansibleop/ansible-operator-overview/step5.md [hunk 2]
```diff
@@ -1,95 +1,161 @@
...
 
-After creating a new operator project using `operator-sdk new --type ansible`,
-the project directory has numerous generated folders and files. The following
+After creating a new operator project the directory has numerous generated folders and files. The following
 table describes a basic rundown of each generated file/directory.
 
 | File/Folders   | Purpose                           |
 | :---           | :--- |
-| deploy | Contains a generic set of Kubernetes manifests for deploying this operator on a Kubernetes cluster. |
+| config | Kustomize YAML definitions required to launch our controller on a cluster. It is the target directory to hold our CustomResourceDefinitions, RBAC configuration, and WebhookConfigurations.
```
Comment (btannous) [inline ansibleop/ansible-operator-overview/step5.md:153]: Do these generate to tables properly?

File: ansibleop/ansible-operator-overview/step6.md [hunk 3]
```diff
@@ -87,13 +86,12 @@ Since we have swapped out the original Role for one from Ansible Galaxy, lets ch
...
   kind: Memcached
   role: /opt/ansible/roles/dymurray.memcached_operator_role
-  
 </pre>
 
 Update this file by running the following command:
 
 ```
-wget -q https://raw.githubusercontent.com/openshift-labs/learn-katacoda/master/ansibleop/ansible-operator-overview/assets/watches.yaml -O /root/tutorial/memcached-operator/watches.yaml
+\cp /tmp/watches.yaml watches.yaml
```
Comment (btannous) [inline ansibleop/ansible-operator-overview/step6.md:94]: `\cp` and other html tags are here

File: ansibleop/ansible-operator-overview/step8.md [hunk 4]
```diff
@@ -43,89 +38,65 @@ of memcached.
...
 spec:
-  size: 4
+  size: 3
 </pre>
 
 Update this file by running the following command:
 
 ```
-wget -q https://raw.githubusercontent.com/openshift-labs/learn-katacoda/master/ansibleop/ansible-operator-overview/assets/cache.example.com_v1alpha1_memcached_cr_updated.yaml -O /root/tutorial/memcached-operator/deploy/crds/cache.example.com_v1alpha1_memcached_cr.yaml
+\cp /tmp/cache_v1alpha1_memcached.yaml config/samples/cache_v1alpha1_memcached.yaml
```
Comment (btannous) [inline ansibleop/ansible-operator-overview/step8.md:90]: `\cp`?

File: ansibleop/ansible-operator-overview/step8.md [hunk 5]
```diff
@@ -43,89 +38,65 @@ of memcached.
...
 ```{{execute}}
 <br>
```
Comment (btannous) [inline ansibleop/ansible-operator-overview/step8.md:92]: html tags.

File: operatorframework/go-operator-podset/step3.md [hunk 6]
```diff
@@ -1,4 +1,15 @@
...
+cat api/v1alpha1/podset_types.go
+```{{execute}}
+
+In Kubernetes, every functional object (with some exceptions, i.e. ConfigMap) includes `spec` and `status`. Kubernetes functions by reconciling desired state (Spec) with the actual cluster state. We then record what is observed (Status). 
+
+Also observe the `+kubebuilder` comment markers found throughout the file. `operator-sdk` makes use of a tool called [controler-gen](https://github.com/kubernetes-sigs/controller-tools) (from the [controller-tools](https://github.com/kubernetes-sigs/controller-tools) project) for generating utility code and Kubernetes YAML. More information on markers for config/code generation can be found [here](https://book.kubebuilder.io/reference/markers.html).
+
+Let's now modify the `PodSetSpec` and `PodSetStatus` of the `PodSet` Custom Resource (CR) at `api/v1alpha1/podset_types.go`
+
 <br>
```
Comment (btannous) [inline operatorframework/go-operator-podset/step3.md:13]: html.

File: operatorframework/go-operator-podset/step3.md [hunk 7]
```diff
@@ -60,31 +69,31 @@ func init() {
 You can easily update this file by running the following command:
 
 ```
-wget -q https://raw.githubusercontent.com/openshift-labs/learn-katacoda/master/operatorframework/go-operator-podset/assets/podset_types.go -O pkg/apis/app/v1alpha1/podset_types.go
+\cp /tmp/podset_types.go api/v1alpha1/podset_types.go
```
Comment (btannous) [inline operatorframework/go-operator-podset/step3.md:72]: `\cp`?

File: operatorframework/go-operator-podset/step4.md [hunk 8]
```diff
@@ -1,8 +1,176 @@
...
+                For(&appv1alpha1.PodSet{}).
+                Owns(&corev1.Pod{}).
+                Complete(r)
+}
+</pre>
+
+You can easily update this file by running the following command:
+
+```
+\cp /tmp/podset_controller.go controllers/podset_controller.go
```
Comment (btannous) [inline operatorframework/go-operator-podset/step4.md:174]: \cp

Comment (btannous) [reply operatorframework/go-operator-podset/step4.md:174]: @madorn i guess these technically work.  i can merge this stuff in if you guys are ok with it or if you want to try to fix them first

**Review by btannous — COMMENTED** [review]
@madorn see comments above, these are mostly minor issues which I'm fine with if you guys are pressed for time and cant get to it yet.  just let me know

File: operatorframework/go-operator-podset/step4.md (hunk 8 above)
Comment (madorn) [reply operatorframework/go-operator-podset/step4.md:174]: @btannous yes, if you don't mind merging now - I can get this stuff cleaned up as soon as next week's workshop is over. thank you for reviewing.

Comment (btannous) [reply operatorframework/go-operator-podset/step4.md:174]: sounds good. thanks!

**Pull request merged**